Source/Renderer/UniformState.js

```diff
@@ -935,6 +961,14 @@ define([
 
         this._frameState = frameState;
         this._temeToPseudoFixed = Transforms.computeTemeToPseudoFixedMatrix(frameState.time, this._temeToPseudoFixed);
+
+        var fov = camera.frustum.fov;
+        var viewport = this._viewport;
+        if (viewport.width > viewport.height * camera.frustum.aspectRatio) {
```

Comment:
This was just a straight port.  Did I misinterpreted something? Here's the original source:
``` c
    if ((double)mainViewport[3] > ((double)mainViewport[2] * p3dScene->pixelAspect))
    {
        pixelSizePerDistance = tan(0.5 * p3dScene->view.m_pCamera->FieldOfView()) * 2.0 / (double)mainViewport[3];
    }
    else
    {
        pixelSizePerDistance = tan(0.5 * p3dScene->view.m_pCamera->FieldOfView()) * 2.0 / (double)mainViewport[2];
    }
```